src/mcp_atlassian/servers/jira.py

```diff
@@ -1653,3 +1654,23 @@ async def batch_create_versions(
             )
             results.append({"success": False, "error": str(e), "input": v})
     return json.dumps(results, indent=2, ensure_ascii=False)
+
+
+@jira_mcp.tool(tags={"jira", "write"})
+async def jira_add_watcher(ctx: Context, issue_key: str, user: str) -> dict[str, Any]:
+    """Add a watcher to a Jira issue."""
+    fetcher = get_jira_fetcher(ctx)
+    watcher_mixin = WatchersMixin(config=fetcher.config)
+    watcher_mixin.jira = fetcher.jira
+    return watcher_mixin.add_watcher(issue_key, user)
```

Comment:
```suggestion
    return await jira.add_watcher(issue_key, user)
```
The tool should call the method directly on the fetcher instance to maintain consistency with all other tools in this file.